src/common/reducers/actions.js

```diff
@@ -3201,9 +3203,10 @@ export function launchInstance(instanceName, forceQuit = false) {
     );
 
     const needsQuote = process.platform !== 'win32';
+    const javaPathWithEnvVar = needsQuote ? resolveEnvVars(javaPath) : javaPath;
```

Comment:
I think this line fails on my platform. Shouldn't it be this?
```suggestion
    const javaPathWithEnvVar = needsQuote ? javaPath : resolveEnvVars(javaPath);
```

Reply:
you are right, my bad